src/components/operations/operation-details/ko/runtime/operation-details.ts

```diff
@@ -86,11 +86,19 @@ export class OperationDetails {
                 operationPath += operation.displayUrlTemplate;
             }
 
+            let requestUrl = "";
+
             if (api.type === TypeOfApi.webSocket) {
-                return `${hostname}${Utils.ensureLeadingSlash(operationPath)}`;
+                requestUrl = `${hostname}${Utils.ensureLeadingSlash(operationPath)}`;
+            } else {
+                requestUrl = `https://${hostname}${Utils.ensureLeadingSlash(operationPath)}`;
+            }
+
+            if (api.apiVersionSet && api.apiVersion && api.apiVersionSet.versioningScheme === "Query") {
```

Comment:
nit: this
`api.apiVersionSet && api.apiVersion && api.apiVersionSet.versioningScheme === "Query"`
could be simplified a bit:
`api.apiVersion && api.apiVersionSet?.versioningScheme === "Query"`